plugins/virt.yaml

```diff
@@ -58,17 +58,23 @@ spec:
   caveats: |
     virt plugin is a wrapper for virtctl originating from the KubeVirt project. In order to use virtctl you will
     need to have KubeVirt installed on your Kubernetes cluster to use it. See https://kubevirt.io/ for details
-  description: |
-    virt plugin is a wrapper for virtctl originating from the KubeVirt project. KubeVirt is a virtualization add-on to
-    Kubernetes, i.e. it enables to run existing virtual machines on Kubernetes clusters. virtctl controls virtual
-    machine related operations on your Kubernetes cluster like connecting to the serial and VNC consoles.
 
     Run
 
       kubectl virt help
```

Comment:
What will happen if you run `kubectl virt`, will it also show the help Text? If so, consider removing this, to keep it as short as possible.

Reply:
Actual output of latest v0.30.0 version looks like this:
```
$ kubectl virt version
Client Version: version.Info{GitVersion:"v0.30.0-49+d358cf085b5a86", GitCommit:"d358cf085b5a86cc4fa516215f8b757a4e61def2", GitTreeState:"clean", BuildDate:"2020-06-09T12:22:50Z", GoVersion:"go1.12.8", Compiler:"gc", Platform:"linux/amd64"}
the server could not find the requested resource
$ kubectl virt
Available Commands:
  console      Connect to a console of a virtual machine instance.
  expose       Expose a virtual machine instance, virtual machine, or virtual machine instance replica set as a new service.
  help         Help about any command
  image-upload Upload a VM image to a DataVolume/PersistentVolumeClaim.
  migrate      Migrate a virtual machine.
  pause        Pause a virtual machine
  rename       Rename a stopped virtual machine.
  restart      Restart a virtual machine.
  start        Start a virtual machine.
  stop         Stop a virtual machine.
  unpause      Unpause a virtual machine
  version      Print the client and server version information.
  vnc          Open a vnc connection to a virtual machine instance.

Use "kubectl virt <command> --help" for more information about a given command.
Use "kubectl virt options" for a list of global command-line options (applies to all commands).
```